assets/js/components/ProjectEditor.js

```diff
@@ -103,15 +120,17 @@ export function ProjectEditor (projectDescriptionCredits, programId, textFields,
 
   this.close = () => {
     $(window).off('popstate', this.popStateHandler)
-    showDefaultTopBarTitle()
+    $(window).on('popstate', this.navigationHandler)
```

Comment:
Should this be in reopenNavigation or does it break things?

Reply:
Moved to reopenNavigation